Week-1/Blastoise/Web_scrapper.py

```diff
@@ -0,0 +1,127 @@
+import requests
+from bs4 import BeautifulSoup as soup
+import json
+import smtplib
+import csv
+import datetime
+from threading import Timer
+# For running code every minute
+def runn_multiple_times():
```

Comment:
This function body is huge. It'll be a nightmare if you come to this code after a while, i.e. maintenance will bug you a lot.

How about breaking the function into small parts? Adding the properties of a phone could be another function altogether and so on.